tensorflow/python/keras/saving/hdf5_format_test.py

```diff
@@ -335,6 +336,46 @@ def test_sequential_weight_loading_group_name_with_incorrect_length(self):
                                  r'element\(s\)\.'):
       hdf5_format.load_weights_from_hdf5_group_by_name(f_model, model.layers)
 
+  @test_util.run_in_graph_and_eager_modes
+  def test_sequential_weight_loading_group_name_with_incorrect_length_with_skip_mismatch(self):
+    if h5py is None:
+      return
+
+    temp_dir = self.get_temp_dir()
+    self.addCleanup(shutil.rmtree, temp_dir)
+    h5_path = os.path.join(temp_dir, 'test.h5')
+
+    num_hidden = 5
+    input_dim = 3
+    num_classes = 2
+    with self.cached_session():
+      ref_model = keras.models.Sequential()
+      ref_model.add(keras.layers.Dense(num_hidden, input_dim=input_dim,
+                                       name='d1'))
+      ref_model.add(keras.layers.Dense(num_classes, name='d2'))
+      ref_model.compile(loss=keras.losses.MSE,
+                        optimizer=keras.optimizers.RMSprop(lr=0.0001),
+                        metrics=[keras.metrics.categorical_accuracy])
+
+      f_ref_model = h5py.File(h5_path, 'w')
+      hdf5_format.save_weights_to_hdf5_group(f_ref_model, ref_model.layers)
+
+      f_model = h5py.File(h5_path, 'r')
+      model = keras.models.Sequential()
+      model.add(keras.layers.Dense(num_hidden, use_bias=False,
+                                   input_dim=input_dim, name='d1'))
+      model.add(keras.layers.Dense(num_classes + 1, name='d2')) #  This shape is modified.
+      model.compile(loss=keras.losses.MSE,
+                    optimizer=keras.optimizers.RMSprop(lr=0.0001),
+                    metrics=[keras.metrics.categorical_accuracy])
+    with warnings.catch_warnings(record=True) as w:
+      warnings.simplefilter("always")
+
+      hdf5_format.load_weights_from_hdf5_group_by_name(f_model, model.layers, skip_mismatch=True)
```

Comment:
Line length here and on line 340 and 367 (also maybe other places)

Reply:
Ah line lengths, fun :)

Do you have a proposal for how to modify line 340? I choose the same name as the unit test it is based on and added `_skip_mismatch`, but it makes the name already be longer than 80 characters.

Reply:
I think the usual way to do this is

```
      hdf5_format.load_weights_from_hdf5_group_by_name(
          f_model, model.layers, skip_mismatch=True)
```